### DISCUSSION TITLE
The Getting-Started wiki is out of date

### DISCUSSION THREAD

**Discussion by deerball** [optimatika/ojAlgo#523] [General, ojAlgo and related code]
https://github.com/optimatika/ojAlgo/wiki/Getting-Started

Will anyone help update it?

**Comment by apete** [comment]
Just to let you know. 

Personally, I have stopped updating the wiki here at GitHub. I focus on the contents at [the ojAlgo web site](https://www.ojalgo.org).

If you, or anyone, is willing to update it, you are more than welcome. In fact I kind of hoped someone would...
